### PR TITLE
Add videojs-overlay w/ git auto-update via single package.json

### PR DESCRIPTION
**Pull request by SelvarajKaruppusamy**
closes: #9826

Pull request for issue: #9826
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): [https://github.com/brightcove/videojs-overlay](https://github.com/brightcove/videojs-overlay)
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/videojs-overlay
 * License and its reference: [Apache 2.0](https://github.com/brightcove/videojs-overlay/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:36
   - Count of stars:84
   - Count of forks:51
 * NPM download stats (optional):
   - Downloads in the last day:231
   - Downloads in the last week:1080
   - Downloads in the last month:3993

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
Hi @SelvarajKaruppusamy 
Thank you for your contribution.
I think this PR is fine. 

May I ask you how do you send this pull request? via GitHub website or use git command?
I think the commit should squash to one commit and rebase to the latest master, if you use git command, we can help you how to do this.
But if you use GitHub website, it can't squash the commit. I can help you squash to one commit.

BTW, it will be helpful to us to let the commit message as followings:
```
Add videojs-overlay w/ git auto-update via single package.json

close #9826
```

Thanks.

**Comment by SelvarajKaruppusamy** [comment]
Hi Samina,

First, Thank you for reaching out to help me.

I sent the pull request via GitHub website, but later I did a sparse clone
of the repository. I would be able to squash it.

Yet I have to,
  1. Pick the new commits to master (another 14 commits)
  2. Rebase the feature branch to the new base
  3. Squash the commits with the above mentioned message

My laptop is considerably slow, so it will take some time to do the above.
Please bare with me.

Thanks,
Selvaraj

**Comment by SelvarajKaruppusamy** [comment]
@sufuf3, @qwe19272375 Sure. I will update the changes soon.

**Comment by PeterDaveHello** [comment]
@sufuf3 any reason not to mention the author in commit message this time?

**Comment by sufuf3** [comment]
@SelvarajKaruppusamy I really sorry that I didn't notice that the commit message should tag the author.
Would you tag the author in the commit message as the followings?
```
Add videojs-overlay w/ git auto-update via single package.json

close #9826, cc @misteroneill, @dmlap, @BrandonOCasey
```
"dmlap" is the author who created this repo.
"misteroneill" and "BrandonOCasey" seem that who maintain this repo recently.

Thank you for your contribution.

**Comment by PeterDaveHello** [comment]
@sufuf3 I think you can directly help revise it this time if everything else is good enough.

**Comment by sufuf3** [comment]
Help revise the commit message about tag authors.

**Comment by SelvarajKaruppusamy** [comment]
@sufuf3  Looks like the commit is revised. So, all good?

**Comment by sufuf3** [comment]
I think this PR is Good. 😄 
Let's wait for the maintainer's response.

cc @PeterDaveHello

File: ajax/libs/videojs-overlay/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "videojs-overlay",
+  "description": "A plugin to display simple overlays during video playback.",
+  "author": "Brightcove, Inc.",
+  "license": "Apache-2.0",
+  "keywords": [
+    "videojs",
+    "videojs-plugin"
```
Comment (PeterDaveHello) [inline ajax/libs/videojs-overlay/package.json]: @sufuf3 @dakshshah96 I think we can add more keywords here next time like `overlay`

**Comment by SelvarajKaruppusamy** [comment]
Thanks for the help guys. Specially @PeterDaveHello, @sufuf3

**Comment by PeterDaveHello** [comment]
@SelvarajKaruppusamy thank you, too.

**Comment by sufuf3** [comment]
@SelvarajKaruppusamy thank you, too. 😃

**Pull request merged**